source/includes/databases-collections.kt

```diff
@@ -0,0 +1,47 @@
+import com.mongodb.ConnectionString
+import com.mongodb.MongoClientSettings
+import com.mongodb.kotlin.client.MongoClient
+import org.bson.Document
+import org.bson.json.JsonWriterSettings
+
+fun main() {
+    val uri = "<connection string URI>"
+    
+    val settings = MongoClientSettings.builder()
+        .applyConnectionString(ConnectionString(uri))
+        .retryWrites(true)
+        .build()
+
+    val client = MongoClient.create(settings)
+
+    // Accesses the "test_database" database
+    // start-access-database
+    val db = client.getDatabase("test_database")
+    // end-access-database
+
+    // Accesses the "test_collection" collection
+    // start-access-collection
+    val collection = db.getCollection<Document>("test_collection")
```

Comment:
No type argument is used 
```suggestion
    val collection = db.getCollection("test_collection")
```